slices/libx11-data.yaml

```diff
@@ -0,0 +1,202 @@
+      /usr/share/X11/locale/pt_BR.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/pt_PT.UTF-8/Compose:
+      /usr/share/X11/locale/pt_PT.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/pt_PT.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/ru_RU.UTF-8/Compose:
+      /usr/share/X11/locale/ru_RU.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/ru_RU.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/sr_RS.UTF-8/Compose:
+      /usr/share/X11/locale/sr_RS.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/sr_RS.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/tatar-cyr/Compose:
+      /usr/share/X11/locale/tatar-cyr/XI18N_OBJS:
+      /usr/share/X11/locale/tatar-cyr/XLC_LOCALE:
+      /usr/share/X11/locale/th_TH.UTF-8/Compose:
+      /usr/share/X11/locale/th_TH.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/th_TH.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/th_TH/Compose:
+      /usr/share/X11/locale/th_TH/XI18N_OBJS:
+      /usr/share/X11/locale/th_TH/XLC_LOCALE:
+      /usr/share/X11/locale/tscii-0/Compose:
+      /usr/share/X11/locale/tscii-0/XI18N_OBJS:
+      /usr/share/X11/locale/tscii-0/XLC_LOCALE:
+      /usr/share/X11/locale/vi_VN.tcvn/Compose:
+      /usr/share/X11/locale/vi_VN.tcvn/XI18N_OBJS:
+      /usr/share/X11/locale/vi_VN.tcvn/XLC_LOCALE:
+      /usr/share/X11/locale/vi_VN.viscii/Compose:
+      /usr/share/X11/locale/vi_VN.viscii/XI18N_OBJS:
+      /usr/share/X11/locale/vi_VN.viscii/XLC_LOCALE:
+      /usr/share/X11/locale/zh_CN.UTF-8/Compose:
+      /usr/share/X11/locale/zh_CN.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/zh_CN.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/zh_CN.gb18030/Compose:
+      /usr/share/X11/locale/zh_CN.gb18030/XI18N_OBJS:
+      /usr/share/X11/locale/zh_CN.gb18030/XLC_LOCALE:
+      /usr/share/X11/locale/zh_CN.gbk/Compose:
+      /usr/share/X11/locale/zh_CN.gbk/XI18N_OBJS:
+      /usr/share/X11/locale/zh_CN.gbk/XLC_LOCALE:
+      /usr/share/X11/locale/zh_CN/Compose:
+      /usr/share/X11/locale/zh_CN/XI18N_OBJS:
+      /usr/share/X11/locale/zh_CN/XLC_LOCALE:
+      /usr/share/X11/locale/zh_HK.UTF-8/Compose:
+      /usr/share/X11/locale/zh_HK.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/zh_HK.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/zh_HK.big5/Compose:
+      /usr/share/X11/locale/zh_HK.big5/XI18N_OBJS:
+      /usr/share/X11/locale/zh_HK.big5/XLC_LOCALE:
+      /usr/share/X11/locale/zh_HK.big5hkscs/Compose:
+      /usr/share/X11/locale/zh_HK.big5hkscs/XI18N_OBJS:
+      /usr/share/X11/locale/zh_HK.big5hkscs/XLC_LOCALE:
+      /usr/share/X11/locale/zh_TW.UTF-8/Compose:
+      /usr/share/X11/locale/zh_TW.UTF-8/XI18N_OBJS:
+      /usr/share/X11/locale/zh_TW.UTF-8/XLC_LOCALE:
+      /usr/share/X11/locale/zh_TW.big5/Compose:
+      /usr/share/X11/locale/zh_TW.big5/XI18N_OBJS:
+      /usr/share/X11/locale/zh_TW.big5/XLC_LOCALE:
+      /usr/share/X11/locale/zh_TW/Compose:
+      /usr/share/X11/locale/zh_TW/XI18N_OBJS:
+      /usr/share/X11/locale/zh_TW/XLC_LOCALE:
```

Comment:
How about the following instead?

```suggestion
      /usr/share/X11/XErrorDB:
      /usr/share/X11/locale/C/**:
      /usr/share/X11/locale/am_ET.UTF-8/**:
      /usr/share/X11/locale/armscii-8/**:
      /usr/share/X11/locale/compose.dir:
      /usr/share/X11/locale/cs_CZ.UTF-8/**:
      /usr/share/X11/locale/el_GR.UTF-8/**:
      /usr/share/X11/locale/en_US.UTF-8/**:
      /usr/share/X11/locale/fi_FI.UTF-8/**:
      /usr/share/X11/locale/georgian-academy/**:
      /usr/share/X11/locale/georgian-ps/**:
      /usr/share/X11/locale/ibm-cp1133/**:
      /usr/share/X11/locale/iscii-dev/**:
      /usr/share/X11/locale/isiri-3342/**:
      /usr/share/X11/locale/iso8859-1/**:
      /usr/share/X11/locale/iso8859-10/**:
      /usr/share/X11/locale/iso8859-11/**:
      /usr/share/X11/locale/iso8859-13/**:
      /usr/share/X11/locale/iso8859-14/**:
      /usr/share/X11/locale/iso8859-15/**:
      /usr/share/X11/locale/iso8859-2/**:
      /usr/share/X11/locale/iso8859-3/**:
      /usr/share/X11/locale/iso8859-4/**:
      /usr/share/X11/locale/iso8859-5/**:
      /usr/share/X11/locale/iso8859-6/**:
      /usr/share/X11/locale/iso8859-7/**:
      /usr/share/X11/locale/iso8859-8/**:
      /usr/share/X11/locale/iso8859-9/**:
      /usr/share/X11/locale/iso8859-9e/**:
      /usr/share/X11/locale/ja.JIS/**:
      /usr/share/X11/locale/ja.SJIS/**:
      /usr/share/X11/locale/ja/**:
      /usr/share/X11/locale/ja_JP.UTF-8/**:
      /usr/share/X11/locale/km_KH.UTF-8/**:
      /usr/share/X11/locale/ko/**:
      /usr/share/X11/locale/ko_KR.UTF-8/**:
      /usr/share/X11/locale/koi8-c/**:
      /usr/share/X11/locale/koi8-r/**:
      /usr/share/X11/locale/koi8-u/**:
      /usr/share/X11/locale/locale.alias:
      /usr/share/X11/locale/locale.dir:
      /usr/share/X11/locale/microsoft-cp1251/**:
      /usr/share/X11/locale/microsoft-cp1255/**:
      /usr/share/X11/locale/microsoft-cp1256/**:
      /usr/share/X11/locale/mulelao-1/**:
      /usr/share/X11/locale/nokhchi-1/**:
      /usr/share/X11/locale/pt_BR.UTF-8/**:
      /usr/share/X11/locale/pt_PT.UTF-8/**:
      /usr/share/X11/locale/ru_RU.UTF-8/**:
      /usr/share/X11/locale/sr_RS.UTF-8/**:
      /usr/share/X11/locale/tatar-cyr/**:
      /usr/share/X11/locale/th_TH.UTF-8/**:
      /usr/share/X11/locale/th_TH/**:
      /usr/share/X11/locale/tscii-0/**:
      /usr/share/X11/locale/vi_VN.tcvn/**:
      /usr/share/X11/locale/vi_VN.viscii/**:
      /usr/share/X11/locale/zh_CN.UTF-8/**:
      /usr/share/X11/locale/zh_CN.gb18030/**:
      /usr/share/X11/locale/zh_CN.gbk/**:
      /usr/share/X11/locale/zh_CN/**:
      /usr/share/X11/locale/zh_HK.UTF-8/**:
      /usr/share/X11/locale/zh_HK.big5/**:
      /usr/share/X11/locale/zh_HK.big5hkscs/**:
      /usr/share/X11/locale/zh_TW.UTF-8/**:
      /usr/share/X11/locale/zh_TW.big5/**:
      /usr/share/X11/locale/zh_TW/**:
```

Reply:
Looks cleaner :)

Reply:
My bad, probably should have sorted before.